lms/djangoapps/experiments/filters.py

```diff
@@ -1,9 +1,15 @@
 import django_filters
 
-from experiments.models import ExperimentData
+from experiments.models import ExperimentData, ExperimentKeyValue
 
 
 class ExperimentDataFilter(django_filters.FilterSet):
     class Meta(object):
         model = ExperimentData
         fields = ['experiment_id', 'key', ]
```

Comment:
That's a good question. Yes. We will need to support filtering by user. This experiment will be the first where we actually read the data. Item added to wiki.